pyxem/signals/diffraction2d.py

```diff
@@ -1093,6 +1092,11 @@ def template_match_with_binary_image(
         pst._copy_signal_all_axes_metadata(self, s)
         return s
 
+    @deprecated(
+        since="0.14",
```

Comment:
Change to 0.15 since the next minor release is 0.15?

```suggestion
        since="0.15",
```